index.html

```diff
@@ -47,119 +54,131 @@ <h4>LET'S CONNECT</h4>
-                    <li><button class="language-buttons" type="button">javaScript</button></li>
-                </ul>
-                <button class="p-button" type="button">See project</button>
+                <div class="column2">
+                    <h1><strong>Multi-Post Stories</strong></h1>
+                    <ul>
+                        <li class="element1">CANOPY</li>
+                        <li><img src="Counter.svg" alt="dot"></li>
+                        <li class="element2">Back End Dev</li>
+                        <li><img src="Counter.svg" alt="dot"></li>
+                        <li class="element2">2015</li>
+                    </ul>
+                    <p>A daily selection of privately personalized reads; 
+                        no accounts or sign-ups required.</p>
+                    <ul class="bottom-list">
+                        <li><button class="language-buttons" type="button">html</button></li>
+                        <li><button class="language-buttons" type="button">css</button></li>
+                        <li><button class="language-buttons" type="button">javaScript</button></li>
+                    </ul>
+                    <button class="p-button" type="button">See project</button>
+                </div>
             </div> 
         </section>
-        <section class="about-me">
-            <h1>About<br> Myself</h1>
-            <p class="description">I’m a software developer! 
-                I can help you build a product, 
-                feature or website. Look through 
-                some of my work and experience! 
-                If you like what you see and have 
-                a project you need coded, 
-                don’t hestiate to contact me.
-            </p>
-            <div class="connect">
-                <h4>LET'S CONNECT</h4>
-            </div>
-            <div class="social">
-                <ul>
-                    <li>
-                        <a href="https://www.facebook.com/emmanuel.cortes.944"><img src="twit.svg" alt="twitter"></a>
-                    </li>
-                    <li>
-                       <a href="www.linkedin.com/in/emmanuel-cortés-padilla-49098214"><img src="linkedin.svg" alt="linkedin"></a> 
-                    </li>
-                    <li>
-                        <a href="#"><img src="m.svg" alt="m"></a>
-                    </li>
-                    <li>
-                        <a href="https://github.com/emmanuelcortes2"><img src="git.svg" alt="git"></a>
-                    </li>
-                    <li>
-                        <a href="#"><img src="peaceandlove.svg" alt="peace"></a>
-                    </li>
-                </ul>
-            </div> 
-            <div>
-                <button class="p-button" type="button">Get my resume</button>
+        <section class="about-me" id="about-me">
+            <div class="column1">
+                <h1><strong>About<br>Myself</strong></h1>
```

Comment:
- [ ] As the previous reviewer suggested, kindly increase the font weight of your **_` About Myself`_** heading to match the Figma design template you chose🤗 and also use the right color specified in the Figma design as your color is currently black.

Your design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188229322-aaa087cf-1f55-4c45-8004-2c726d9e87a2.png)

Figma design 👇🏽 

![image](https://user-images.githubusercontent.com/63473584/188229495-2a38a29d-b1e6-4509-bd32-b2f5399f6dc5.png)